docs/diagrams/UiClassDiagram.puml

```diff
@@ -33,28 +37,39 @@ UiManager -down-> MainWindow
 MainWindow --> HelpWindow
 MainWindow *-down-> CommandBox
 MainWindow *-down-> ResultDisplay
-MainWindow *-down-> PersonListPanel
-MainWindow *-down-> StatusBarFooter
+MainWindow *-down-> AssignmentListPanel
+MainWindow *-down-> UpcomingTaskListPanel
+MainWindow *-down-> ReminderListPanel
 
-PersonListPanel -down-> PersonCard
+AssignmentListPanel -down-> AssignmentCard
+UpcomingTaskListPanel -down-> UpcomingLessonCard
+UpcomingTaskListPanel -down-> UpcomingAssignmentCard
+ReminderListPanel -down-> ReminderCard
 
 MainWindow -left-|> UiPart
 
 ResultDisplay --|> UiPart
 CommandBox --|> UiPart
-PersonListPanel --|> UiPart
-PersonCard --|> UiPart
-StatusBarFooter --|> UiPart
+AssignmentListPanel --|> UiPart
+AssignmentCard --|> UiPart
 HelpWindow -down-|> UiPart
+ReminderCard --|> UiPart
+ReminderListPanel --|> UiPart
+UpcomingAssignmentCard --|> UiPart
+UpcomingLessonCard --|> UiPart
+UpcomingTaskListPanel --|> UiPart
+
+UpcomingLessonCard ..> Model
+AssignmentCard ..> Model
+UpcomingAssignmentCard ..> Model
+ReminderCard ..> Model
```

Comment:
These arrows seem to be combined together in the image. Is there a way to rearrange?

Reply:
I tried to rearrange it but didn't manage to. Can you help me rearrange the arrows after this is merged?